transitions/index.js

```diff
@@ -75,7 +76,7 @@ const processChange = (change, callback) => {
   if (!change) {
     return callback();
   }
-  db.medic.get(change.id, (err, doc) => {
+  utils.hydrateDoc(db, change.id, (err, doc) => {
```

Comment:
Wondering if it would be cleaner if hydrateDoc took an id and a list of docs (contacts) and keep the db query out of that function?